cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/PhaseTapChangerConversion.java

```diff
@@ -100,6 +103,7 @@ public boolean valid() {
         return true;
     }
 
+    @Deprecated
```

Comment:
Not necessary to deprecate the methods when the class is deprecated.

Reply:
Done